reward/staking_info_test.go

```diff
@@ -390,3 +392,102 @@ func TestConsolidatedStakingInfo(t *testing.T) {
+		KFFAddr               common.Address   `json:"PoCAddr"` // PoCAddr -> KFFAddr from v1.10.2
+		UseGini               bool             `json:"UseGini"`
+		Gini                  float64          `json:"Gini"`
+		CouncilStakingAmounts []uint64         `json:"CouncilStakingAmounts"`
+	}
+
+	oldInfo := oldStakingInfo{
+		2880,
+		[]common.Address{
+			common.HexToAddress("0x159ae5ccda31b77475c64d88d4499c86f77b7ecc"),
+			common.HexToAddress("0x181deb121304b0430d99328ff1a9122df9f09d7f"),
+			common.HexToAddress("0x324ec8f2681cd73642cc55057970540a1f4393e0"),
+			common.HexToAddress("0x11191029025d3fcd21001746f949b25c6e8435cc"),
+		},
+		[]common.Address{
+			common.HexToAddress("0x70e051c46ea76b9af9977407bb32192319907f9e"),
+			common.HexToAddress("0xe4a0c3821a2711758306ed57c2f4900aa9ddbb3d"),
+			common.HexToAddress("0xf3ba3a33b3bf7cf2085890315b41cc788770feb3"),
+			common.HexToAddress("0x9285a85777d0ae7e12bee3ffd7842908b2295f45"),
+		},
+		[]common.Address{
+			common.HexToAddress("0xd155d4277c99fa837c54a37a40a383f71a3d082a"),
+			common.HexToAddress("0x2b8cc0ca62537fa5e49dce197acc8a15d3c5d4a8"),
+			common.HexToAddress("0x7d892f470ecde693f52588dd0cfe46c3d26b6219"),
+			common.HexToAddress("0xa0f7354a0cef878246820b6caa19d2bdef74a0cc"),
+		},
+		common.HexToAddress("0x673003e5f9a852d3dc85b83d16ef62d45497fb96"),
+		common.HexToAddress("0x576dc0c2afeb1661da3cf53a60e76dd4e32c7ab1"),
+		false,
+		-1,
+		[]uint64{5000000, 5000000, 5000000, 5000000},
+	}
+
+	newInfo := StakingInfo{
+		oldInfo.BlockNum,
+		[]common.Address{
+			common.HexToAddress("0x159ae5ccda31b77475c64d88d4499c86f77b7ecc"),
+			common.HexToAddress("0x181deb121304b0430d99328ff1a9122df9f09d7f"),
+			common.HexToAddress("0x324ec8f2681cd73642cc55057970540a1f4393e0"),
+			common.HexToAddress("0x11191029025d3fcd21001746f949b25c6e8435cc"),
+		},
+		[]common.Address{
+			common.HexToAddress("0x70e051c46ea76b9af9977407bb32192319907f9e"),
+			common.HexToAddress("0xe4a0c3821a2711758306ed57c2f4900aa9ddbb3d"),
+			common.HexToAddress("0xf3ba3a33b3bf7cf2085890315b41cc788770feb3"),
+			common.HexToAddress("0x9285a85777d0ae7e12bee3ffd7842908b2295f45"),
+		},
+		[]common.Address{
+			common.HexToAddress("0xd155d4277c99fa837c54a37a40a383f71a3d082a"),
+			common.HexToAddress("0x2b8cc0ca62537fa5e49dce197acc8a15d3c5d4a8"),
+			common.HexToAddress("0x7d892f470ecde693f52588dd0cfe46c3d26b6219"),
+			common.HexToAddress("0xa0f7354a0cef878246820b6caa19d2bdef74a0cc"),
+		},
+		common.HexToAddress("0x673003e5f9a852d3dc85b83d16ef62d45497fb96"),
+		common.HexToAddress("0x576dc0c2afeb1661da3cf53a60e76dd4e32c7ab1"),
+		false,
+		-1,
+		[]uint64{5000000, 5000000, 5000000, 5000000},
+	}
+
+	// initialize StakingManager
+	_ = NewStakingManager(newTestBlockChain(), newDefaultTestGovernance(), nil)
+
+	for _, info := range []interface{}{oldInfo, newInfo} {
+		// reset database
+		stakingManager.stakingInfoDB = database.NewMemoryDBManager()
```

Comment:
```suggestion
		SetTestStakingManagerWithDB(database.NewMemoryDBManager())
```
Do not modify the existing instance. Create a new one in each loop.

Reply:
Applied. Thanks!